### ISSUE TITLE
Use Host Server Email address for notifications

### ISSUE THREAD

**Issue by RoiJS** [RoiJS/Reserbiz#29]
On the hosting server, I  prepared new email addresses and these are the following:
* info@reserbiz.com
* noreply@reserbiz.com
* developer@reserbiz.com

Now, the goal is to use these following email address on features that sends email notification on the application/infrastructure. 

TODO:
- [x] info@reserbiz.com -> Demo and regular database creation
- [x] noreply@reserbiz.com -> Notification when sending statement of account to tenants
- [x] developer@reserbiz.com -> Notification every after auto-generation of statement of accounts and penalties and removing expired refresh tokens.
- [x] Makes sure to BCC the email to reserbiz@gmail.com
- [x] Test and verify

**Comment by RoiJS** [comment]
There is problem sending email after auto generating statement of accounts and penalties, the email are basically throttled by the email server. According to this link [https://www.smarterasp.net/support/kb/a1533/sender-is-throttled-error-message.aspx?KBSearchID=874295](https://www.smarterasp.net/support/kb/a1533/sender-is-throttled-error-message.aspx?KBSearchID=874295), I can only send 50 messages per hour. 

On my email server, I now set the setting to unlimited but emails are still throttling. I now sent a support ticket to hosting provider. Still waiting for their response.

**Comment by RoiJS** [comment]
I got a response from the support now. According to them, my domain is set to only have 50 outbound emails per hour, so even if I set the setting to have unlimited outbound emails per hour, this does not works. I have to avail additional add-ons to be able to send more than 50 emails per hour.

As I assess it, I don't think I would need have this feature at least for now so I will stick with my current domain setup. We will decide later if I need avail the add-on next time. 

Moving this issue to close now.
